shared/controller_base.rb

```diff
@@ -79,5 +80,34 @@ def setup_common_controller_configuration(is_called_during_reload: false)
       #   }
       # end
     end
+
+    protected
+
+    # View helper
+    #
+    # @return [String]
+    def new_uuid
+      SecureRandom.uuid
```

Comment:
I think this should live as API in a service, that ultimately asks the data_store, that would be the best move. This is because the backing data store might have a different idea about what a UUID should be. For example, a database might actually prefer auto-incrementing int64, so if we move this to datastore, it can do what it needs.

Reply:
Oh, oops. I forgot to remove this method. I'm not using it anymore. I really like the idea you suggested though. Would it be possible to do that in a separate PR?

Reply:
👍